src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -383,11 +383,18 @@ public Task<Channel> GetChannelAsync(string channel)
         /// <returns>Latest build of <paramref name="repoUri"/> on channel <paramref name="channelId"/>,
         /// or null if there is no latest.</returns>
         /// <remarks>The build's assets are returned</remarks>
-        public Task<Build> GetLatestBuildAsync(string repoUri, int channelId)
+        public async Task<Build> GetLatestBuildAsync(string repoUri, int channelId)
         {
             CheckForValidBarClient();
-            return _barClient.GetLatestBuildAsync(repoUri: repoUri,
-                                                  channelId: channelId);
+            try
+            {
+                return await _barClient.GetLatestBuildAsync(repoUri: repoUri,
+                                                      channelId: channelId);
```

Comment:
Nit: this could be a single line or move `repoUri: repoUri` to its own line as well